### ISSUE TITLE
Transform error in vue2.7

### ISSUE THREAD

**Issue by yandixuan** [unplugin/unplugin-turbo-console#32]
### Describe the bug

https://github.com/unplugin/unplugin-turbo-console/blob/764294cd349409e689fae17755b1753b3e3a5ac7/src/core/transform/compilers/vue.ts#L15-L20

The `descriptor` will be `undefined`

### Reproduction

None

### System Info

```Shell
System:
    OS: Windows 10 10.0.19045
    CPU: (8) x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
    Memory: 5.94 GB / 15.89 GB
  Binaries:
    Node: 20.12.2 - ~\AppData\Local\fnm_multishells\10544_1713747110239\node.EXE
    npm: 10.5.0 - ~\AppData\Local\fnm_multishells\10544_1713747110239\npm.CMD
    pnpm: 8.15.7 - ~\AppData\Local\fnm_multishells\10544_1713747110239\pnpm.CMD
    bun: 1.1.3 - D:\Scoop\shims\bun.EXE
  Browsers:
    Edge: Chromium (123.0.2420.97)
    Internet Explorer: 11.0.19041.1566
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by yuyinws** [comment]
Could you provide a minimal Vue SFC file which case this error?

**Comment by yandixuan** [comment]
https://github.com/yandixuan/unplugin-console-demo

Run and watch the console

**Comment by yuyinws** [comment]
Thanks for your report.
The reason for this problem is `vue/compiler-sfc` provides different API in 2.7 and 3+ versions.
It should be fixed on [v1.8.3 (release)](https://github.com/unplugin/unplugin-turbo-console/releases/tag/v1.8.3)
